SU-HSDO/suhumsci
# Provision Economics and IT-Humsci site

# READY FOR REVIEW

# Summary
Includes the code changes to two new multisites: Economics and IT Humsci. Also includes some docs changes that we discovered needed updating throughout this process.

# Need Review By (Date)
11/25

# Urgency
Low

# Steps to Test
1. Check that the two new multisite directories exist and that they follow our naming conventions.
2. Check that drush aliases have been added for the two new sites and that they are correct.
    - You may be able to test both of these things by creating a local database for each of these sites and and running `drush @[newsite].local si config_installer -y`, to see if it installs successfully.
3. Browse to the following URLs to confirm that the pages load with valid SSL certs (they will redirect to the default site for now):
    - https://economics-dev.stanford.edu
    - https://it-humsci-dev.stanford.edu
    - https://economics-stage.stanford.edu
    - https://it-humsci-stage.stanford.edu
    - https://economics-prod.stanford.edu
    - https://it-humsci-prod.stanford.edu
4. Review the docs changes to ensure they make sense.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
